src/lineup/internal/LineUpPanelActions.ts

```diff
@@ -114,14 +122,27 @@ export default class LineUpPanelActions extends EventHandler {
 
   set collapse(value: boolean) {
     this.node.classList.toggle('collapsed', value);
+
+    // When this.options.enableSidePanel === 'top' `this.collapsed=true` gets called. Attempting to open the PanelTab produces an error.
```

Comment:
When you have the `enableSidePanel='top'`  `this.collapse `gets set one time.
The problem is that i want the PanelTab to react accordingly when the SidePanel gets collapsed, apart from  when `enableSidePanel='top'` because there is no PanelTabs in top mode.
Running ` this.tabContainer.hideCurrentTab();` throws an error because` this.tabContainer` doesn't exist.

Reply:
I changed the implementation and added a NullTabContainer, which has no functionality, for the top mode. This avoids these ugly if-else conditions.